src/Uno/Prism.Uno/Services/Dialogs/DialogService.WinUI.cs

```diff
@@ -0,0 +1,140 @@
+﻿using Prism.Common;
```

Comment:
this can drop the WinUI extension since we should be excluding the WPF implementation